docs/fragments/contact.md

```diff
@@ -18,7 +18,7 @@ Subtitle of the form. This is displayed in the section shows the form.
 ### post_url *string* *Either **post_url** or **email** should be provided*
 If set, submitted data would be sent to this url.
 ### email *string* *Either **post_url** or **email** should be provided*
-If set and there is no **post_url** provided, then the form data would be sent to `https://formspree.io/[email]`. This is the default behavior and can be overriden using the `post_url` variable.
+If set and there is no **post_url** provided, then the form data would be sent to `https://formspree.io/[emsail]`. This is the default behavior and can be overriden using the `post_url` variable.
```

Comment:
Added typo.

Reply:
This page is heavily damaged. The variables and their types and descriptions are in the same line. Also I think this page will be removed when new docs page goes out along with v0.15. I fixed the typo, should I fix the problems as well for now or should we wait and delete it later? BTW I'm not sure why I changed email in here but that was an accident.

Reply:
Let's fix the specific docs in another PR.

Typo fix aka no change seems like a good thing to do here.